reference-controllers/rabbitmq-controller/src/main.rs

```diff
@@ -201,8 +188,8 @@ async fn reconcile_server_config_map(
                 api_resource,
                 rabbitmq.metadata.namespace.as_ref().unwrap(),
                 &server_config_name,
-                "updatedAt",
-                &Utc::now().to_rfc3339(),
+                "lastVersion",
```

Comment:
Can you rename this to lastConfigMapVersion

Reply:
I just realized that this is in `reconcile_server_config_map` -- why do we need to update the configmap's annotation?